addons/common/functions/fnc_registerItemReplacement.sqf

```diff
@@ -20,15 +20,23 @@
 params [["_oldItem", "", [0,""]], ["_newItems", "", ["", []]], ["_replaceInherited", false, [false]]];
 TRACE_3("registerItemReplacement",_oldItem,_newItems,_replaceInherited);
 
-
 // Setup on first run
 if (isNil QGVAR(itemReplacements)) then {
-    GVAR(itemReplacements) = [] call CBA_fnc_createNamespace;
+    GVAR(itemReplacements) = createHashMap;
     GVAR(inheritedReplacements) = [];
     GVAR(oldItems) = [];
     ["loadout", LINKFUNC(replaceRegisteredItems)] call CBA_fnc_addPlayerEventHandler;
 };
 
+// Get config case
+if (_oldItem isEqualType "") then {
+    private _configCase = _oldItem call FUNC(getConfigName);
+
+    if (_configCase != "") then {
+        _oldItem = _configCase;
+    };
```

Comment:
Should we keep this or throw an error instead?

Reply:
If `""` is returned, it means the item doesn't exist in `"CfgWeapons", "CfgMagazines", "CfgGlasses", "CfgVehicles", "CfgVoice", "CfgUnitInsignia"`, which should cover any replaceable item possible.
I was thinking of throwing the error and exiting immediately.

Reply:
Changed it.